spec/features/recipe_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
```

Comment:
- I think it's better to add a foreign key from the foods to the recipe_foods. The tests are failing without this foreign key.